src/renderer/vt/paint.cpp

```diff
@@ -479,8 +479,7 @@ using namespace Microsoft::Console::Types;
     RETURN_IF_FAILED(_MoveCursor(coord));
 
     // Write the actual text string
-    std::wstring wstr = std::wstring(unclusteredString.data(), cchActual);
-    RETURN_IF_FAILED(VtEngine::_WriteTerminalUtf8(wstr));
+    RETURN_IF_FAILED(VtEngine::_WriteTerminalUtf8({ _bufferLine.data(), cchActual }));

```

Comment:
is it not possible to pass the wstring directly as a view? bah.

were we literally making a whole copy of the wstring to a wstring?

Reply:
I don't know why the copy was there. It's probably because the `_WriteTerminalUtf8` function converted to using `std::wstring_view` from `std::wstring` at a point in time after this paint function was last updated and this became a mechanical replace.

As for why we don't pass `_bufferLine` directly, the `cchActual` may be less than the length of the full buffer depending on trimming calculations performed above.